js/models/stock.js

```diff
@@ -104,3 +106,18 @@ var data = [
 ];
 
 // define your Stock model here
+var Stock = function(symbol){
+  // var currentStock = symbol soemthing something
+  // search for symbol in var data
+  // something like data([Symbol: symbol])
+  // this.price
+  // this.companyName
+  // this.numShares
+  // this.totalValue
```

Comment:
this pseudocode looks good! how can you assign the value of `this.price` given the variable `symbol`